conan/tools/meson/meson.py

```diff
@@ -30,9 +30,9 @@ def configure(self, source_folder=None):
 
         cmd = "meson setup"
         if cross_building(self._conanfile):
-            cmd += ' --cross-file "{}"'.format(MesonToolchain.cross_filename)
+            cmd += ' --cross-file "{}"'.format(os.path.join(self._conanfile.install_folder, MesonToolchain.cross_filename))
```

Comment:
Actually, it has been merged to develop branch a new approach: Generated files will be located in ``conanfile.generators_folder``.  Please use that for locating the file.

Reply:
How do I go about configuring the generators_folder? What is the difference with the install_folder?

Now I have the following in my `conanfile.py` I don't want that my files are default in the project root
`    def configure(self):
        self.folders.generators = "install"`

Reply:
pkg_config is still generated in the install_folder.
So I changed some of the folders.

https://docs.conan.io/en/latest/creating_packages/toolchains.html contains some outdated info.
The problem is in the MesonToolchain. It puts the files into the current folder (for 'new generators' this is the generators folder). However if the generator is used according to the docs the folder is the folder where the conan is executed.

> I suggest adapting the MesonToolchain to always generate the items in the generators_folder.